core/modules/wsched/ChunkTasksQueue.h

```diff
@@ -97,9 +97,9 @@ class ChunkTasks {
     };
 
 private:
-    int _chunkId;          ///< Chunk Id for all Tasks in this instance.
-    bool _active{false};   ///< True when this is the active chunk.
-    bool _resourceStarved; ///< True when advancement is prevented by lack of memory.
+    int _chunkId;                    ///< Chunk Id for all Tasks in this instance.
+    bool _active{false};            ///< True when this is the active chunk.
+    bool _resourceStarved{false};   ///< True when advancement is prevented by lack of memory.
```

Comment:
Out of curiosity, do we have any standards on ho members should be initialized. I've now see people use all the variants allowed in C++11. Perhaps some guidance on which forms are better than others and in what contexts they should be used.